apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.ExecutableCommand.cs

```diff
@@ -258,7 +266,11 @@ private async Task<int> ExecuteDdlAsync(CancellationToken cancellationToken)
                             ProtoDescriptors = CommandTextBuilder.ProtobufDescriptors?.ToByteString() ?? ByteString.Empty,
                         };
                         var response = await databaseAdminClient.CreateDatabaseAsync(request).ConfigureAwait(false);
-                        response = await response.PollUntilCompletedAsync().ConfigureAwait(false);
+                        operation = response.RpcMessage;
+                        if (pollUntilCompleted)
+                        {
+                            response = await response.PollUntilCompletedAsync().ConfigureAwait(false);
+                        }
                         if (response.IsFaulted)
```

Comment:
This check should happen before you get the operation.

Reply:
Done